pandunia-loge.csv

```diff
@@ -1750,61 +1746,60 @@ jene loka||||birthplace|lieu de naissance|lugar de nacimiento|terra natal|мес
+joven fem||||girl||niña|||||女の子||||||||knabino|tyttö (nuori neito)|dziewczyna
+joven man||||boy||niño|||||男の子||||||||knabo|poika (nuori mies)|chłopak
+joven muku||||sapling|pousse|pimpollo||саженец|||若木||||||||||młode drzewo
+joven ta||||youth (young age)||juventud|||||||||||||juneco|nuoruus|młodość
 juda||||judge||juzgar||||||||||||||tuomita|sądzić (osądzić)
 judau||||judo||yudo|||||||||||||ĵudo|judo|judo
 jude||||judgment (sentence)||fallo (juicio, criterio)|||||||||||||juĝo|tuomio|wyrok
-juder|||eng:judge, fra:juge, spa:juez, por:juiz, hin:जज (jaj), ben:জজ (jôj), swa:jaji + rus:судья (sud'ya)|judge (referee)|juje|juez (árbitro, réferi)|juiz|судья|||||||||||juĝisto|tuomari|sędzia
-judi||||judicial||||||||||||||||oikeudellinen|
+jude ja|||eng:judge, fra:juge, spa:juez, por:juiz, hin:जज (jaj), ben:জজ (jôj), swa:jaji + rus:судья (sud'ya)|judge (referee)|juje|juez (árbitro, réferi)|juiz|судья|||||||||||juĝisto|tuomari|sędzia
+jude di||||judicial||||||||||||||||oikeudellinen|
 jui|||zho:主义 (zhǔyì), yue:主義 (zyu2 ji6), jpn:主義 (shugi), kor:주의 (juui), vie:chủ nghĩa|ideology (principle, doctrine, -ism)||ideología (doctrina, -ismo)||идеология|||主义|主義|||||||ismo|oikeudellinen (aate, ismi, ideologia, uskonto, jne)|ideologii
 jul|unomete|J||joule (J)|joule (J)|joule (J)|joule (J)|джоуль (J)||焦耳 (J)|ジュール (J)|줄 (J)||जौल (J)||joule (J)||jul (J)|ĵulo (J)|joule (J)|dżul (J)
-jumle|||ara:(jumla), tur:cümle, hin:जुमला (jumlā), zho:句 (jù)|sentence (phrase)||oración||фраза||句子|||||||||||zdanie (fraza)
+jumla|||ara:(jumla), tur:cümle, hin:जुमला (jumlā), zho:句 (jù)|sentence (phrase)||oración||фраза||句子|||||||||||zdanie (fraza)
 jung|||zho:中 (zhōng), yue:中 (zung), kor:중 (jung), jpn:中 (chuu-)|middle (center)|centre|centro|centro|центр|مَرْكَز|中心|中心|중심|trung tâm|केंद्र|কেন্দ্র|pusat|kati (senta)|merkez|centro|keskus (keskikohta)|środkowy, centralny
-junga||||center (concentrate)||centrar (concentrar)||||||||||||||keskittää|centrować, skupić, skupiać
-Jungafrika|desha|CF||Central African Republic||República Centroafricana||||||||||||||Keski-Afrikan Tasavalta|Republika Środkowoafrykańska
-jungi||||central (middle)||central|||||||||||||||centralny (środkowy)
-jungofon||||vowel|voyelle|vocal|vogal|гласный||||보컬 부분||||vokal|||vokalo|vokaali|samogłoska
-jungoshulkan||||secondary school|école secondaire|escuela secundaria|ensino secundário|средняя школа||中学|||trường trung học|||sekolah menengah|||gimnazio|yläaste ja lukio|
-jungu||||to be centered (amid, in the middle)||centrarse (en el centro de)||||||||||||||keskittyä (olla keskellä)|być wycentrowanym, być w środku
+fa jung||||center (concentrate)||centrar (concentrar)||||||||||||||keskittää|centrować, skupić, skupiać
+Jung Afrika Komun krati|desha|CF||Central African Republic||República Centroafricana||||||||||||||Keski-Afrikan Tasavalta|Republika Środkowoafrykańska
```

Comment:
The last word should begin with a capital letter too.

This is what I have been talking about! :D Using capital letters is a mess.